dbpedia/dbPedia_Composers.js

```diff
@@ -123,14 +150,14 @@ function getResults(queryURL, callback) {
             //release
             var release = [];
             if ($('a[rel="dbp:artist"]').attr('href') && $('a[rel="dbp:artist"]').attr('href') != "") {
-                $('a[rel="dbp:artist"]').each(function(index) {
+                $('a[rel="dbp:artist"]').each(function (index) {
                     var rel = replaceURLAndUnderscore($(this).attr('href'));
                     release.push(rel);
                 });
             }
 
             if ($('a[rel="dbo:artist"]').attr('href') && $('a[rel="dbo:artist"]').attr('href') != "") {
```

Comment:
Every selector is used **at most** 3 times, but you're still right. I will adapt all dbpedia scripts.